docker/server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6
+
+RUN true \
+	&& apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		git vim-tiny \
+	&& apt-get clean \
+	&& true
+
+RUN true \
+	&& git clone https://github.com/kiwicom/iwant-bot.git /iwant-bot-cloned \
+	&& ln -s /iwant-bot-cloned /iwant-bot \
+	&& (cd iwant-bot && git checkout docker-support) \
+	&& pip install -r iwant-bot/requirements.txt \
+	&& true
```

Comment:
I am now thinking of two Docker images - one for repeatable builds (i.e. it would download releases from GitHub) and one for development (here I find a shared volume more flexible than copying a directory with source files). Let's discuss this in the upcoming mentoring session.